pkg/network/tracer/event.go

```diff
@@ -376,6 +390,30 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
+	initfunc := func() {
+		procfsPath := "/proc"
+		if config.Datadog.IsSet("procfs_path") {
+			procfsPath = config.Datadog.GetString("procfs_path")
+		}
+
+		ephemeralIntPair = sysctl.NewIntPair(procfsPath, "net/ipv4/ip_local_port_range", time.Hour)
+	}
+	initEphemeralIntPair.Do(initfunc)
+
+	low, hi, err := ephemeralIntPair.Get()
+	if nil == err {
+		ephemeralLow = uint16(low)
+		ephemeralHigh = uint16(hi)
+	}
+	if ephemeralLow == 0 || ephemeralHigh == 0 {
```

Comment:
Can we include `err != nil ||` to this check? That makes it more explicit as to what the error behavior is, rather than relying upon zeroes being returned in the error case.